controlpanel/frontend/static/components/app-logs/macro.html

```diff
@@ -1,4 +1,15 @@
-{% macro app_logs(app) -%}
+{% macro app_logs(app) %}
+  <div class="app-logs">
+    <pre>{% for entry in app.get_logs() -%}
+<span class="timestamp">{{ entry.timestamp.strftime('%Y-%m-%d %H:%M:%S') }}:</span> <span class="message">{{ entry.message }}</span>
+{% endfor %}</pre>
+  </div>
+  <p class="govuk-body">
+    <a href="{{ kibana_link(app) }}">View logs in Kibana</a>
+  </p>
+{% endmacro %}
+
+{% macro kibana_link(app) -%}
 {% filter replace(" ", "") -%}
 {% filter replace("\n", "") -%}
 https://kibana.services.{{env}}.mojanalytics.xyz/app/kibana#/discover?_g=(
```

Comment:
maybe we should start parameterising the domain now because it's going to change soon

Reply:
Good point